include/cpr/parameters.h

```diff
@@ -22,7 +22,7 @@ class Parameters {
     Parameters() = default;
     Parameters(const std::initializer_list<Parameter>& parameters);
 
-    void AddParameter(const Parameter& parameter, const CurlHolder& holder);
+    void AddParameter(const Parameter& parameter, const CurlHolder& holder = CurlHolder());
```

Comment:
I'm not a fan of a default argument here for `holder` since creating a new `CurlHolder()` object can be rather expensive.
We should force users to reuse theirs.